vcsclient/vcsclient.go

```diff
@@ -110,6 +111,12 @@ type VcsClient interface {
 	// pullRequestID  - Pull request ID
 	AddPullRequestComment(ctx context.Context, owner, repository, content string, pullRequestID int) error
 
+	// ListPullRequestComments get all comments assigned to a pull request.
```

Comment:
The following comment also applies to other lines in this file - 
get --> gets
In other places in this file - 
add --> adds
If needed, let's apply this for other verbs that needs be fixed.